### PR TITLE
Fix meta split tests

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

**Comment by ngaut** [comment]
PTAL @disksing @morefreeze

**Comment by disksing** [comment]
LGTM

File: src/storage/mvcc/txn.rs [hunk 1]
```diff
@@ -375,6 +375,12 @@ mod tests {
         must_rollback_then_get(engine.as_ref(), b"x", 15, b"x5");
     }
 
+    // make sure meta version could never catch up with key version(time stamp)
```
Comment (morefreeze) [inline src/storage/mvcc/txn.rs]: `time stamp`? `timestamp`?

**Comment by morefreeze** [comment]
LGTM

**Pull request merged**